src/themes/theme.js

```diff
@@ -42,9 +42,30 @@ export const theme = createTheme({
         fontSize: '4rem',
       },
     },
+    h3: {
```

Comment:
I like how the styles for the headers are in the theme, and you are using the `Typography` component to render them. This is a much nicer approach than using global styles for the header tags 🎉